models/topics.models.js

```diff
@@ -34,10 +35,30 @@ exports.fetchArticles = () => {
   LEFT JOIN comments
     ON articles.article_id = comments.article_id
   GROUP BY articles.author, title, articles.article_id, topic, articles.created_at, articles.votes, article_img_url
-  ORDER BY articles.created_at
-  `
-  return db.query(stringQuery)
-  .then(({rows}) => {
-    return rows
-  })
-}
\ No newline at end of file
+  ORDER BY articles.created_at DESC
+  `;
+  return db.query(stringQuery).then(({ rows }) => {
+    return rows;
+  });
+};
+
+exports.fetchCommentsByArticleId = (article_id) => {
+  const stringQuery = `
+  SELECT comment_id, comments.votes, comments.created_at, comments.author, comments.body, comments.article_id
+  FROM articles
+  LEFT JOIN comments
+    ON articles.article_id = comments.article_id
+  WHERE comments.article_id = $1
+  `;
+  const promiseArray = [];
+  promiseArray.push(db.query(stringQuery, [article_id]));
+  if (article_id !== undefined) {
```

Comment:
Shouldn't need this if statement 🙂. You should get an sql error if you query with undefined ???